src/screens/DateFilterScreen/DateRangePickerDay.test.js

```diff
@@ -86,14 +85,17 @@ describe("dateRangePickerDay", () => {
 
     const output = render({
       date: date(2018, 7, 12),
-      marking: {},
-      state: "disabled"
+      marking: {}
     });
```

Comment:
I recommend mocking the now function in this test so that it is clear that rendering a date before the one returned by now results in a disabled Day component

Reply:
It is mocked, I found an implementation in the same file so thought it best to keep the style consistent. The mock is within the test body :)

Reply:
Apologies, I just saw this was mocked :)